File: experiment_jp_test/j2pyrawpy/while__0__SHORTEST_POSSIBLE_COMBINATION_TWO_STRINGS.py
#!/usr/bin/env python
""" generated source for module while__0__SHORTEST_POSSIBLE_COMBINATION_TWO_STRINGS """
class X(object):
    """ generated source for class X """
    @classmethod
    def printSuperSeq(cls, a, b):
        """ generated source for method printSuperSeq """
        m = len(a)
        n = len(b)
        dp = [None]*m + 1
        res = " "
        i = m
        j = n
        while i > 0:
            res = a.charAt(i - 1) + res
            i -= 1
        while j > 0:
            res = b.charAt(j - 1) + res
            j -= 1
        print res

File: experiment_jp_test/j2pyraw/while__0__SHORTEST_POSSIBLE_COMBINATION_TWO_STRINGS.java
class X { static void printSuperSeq ( String a , String b ) { int m = a . length ( ) , n = b . length ( ) ; int [ ] [ ] dp = new int [ m + 1 ] [ n + 1 ] ; String res = " " ; int i = m , j = n ; while ( i > 0 ) { res = a . charAt ( i - 1 ) + res ; i -- ; } while ( j > 0 ) { res = b . charAt ( j - 1 ) + res ; j -- ; } System . out . println ( res ) ; }
 }